.github/workflows/validate-submodule-pull-request.yml

```diff
@@ -28,17 +28,11 @@ jobs:
     steps:
       - name: Add Jira Issue Code to Pull Request
```

Comment:
Update the name of this step and the name of the matching step in the action too.